packages/server/src/authenticator/index.ts

```diff
@@ -0,0 +1,6 @@
+import { authPackage } from './authPackage';
+import { authPrivacy } from './authPrivacy';
+import { authProperty } from './authProperty';
+import { authByAuthType } from './authByAuthType';
+
+export { authPackage, authPrivacy, authProperty, authByAuthType };
```

Comment:
I think it would be better to export immediately rather than export after import.
like this: 
`export * from './authPackage';`